modules/core/src/main/java/org/apache/ignite/configuration/ClientConfiguration.java

```diff
@@ -529,11 +531,17 @@ public ClientConfiguration setTransactionConfiguration(ClientTransactionConfigur
     }
 
     /**
+     * Partition awareness functionality helps to avoid an additional network hop in the following scenarios:
+     * <ul>
+     *     <li>1. Single-key operations API, like put(), get(), etc. However, the functionality has no effect on those
+     *     operations within explicit transactions {@code ClientTransactions#txStart()}.</li>
```

Comment:
here and below, let's replace @code with @link, and then import required dependencies, to make it possible to navigate for mentioned functions

Reply:
thank you, I will correct